src/views/v2/rsvp/ari/sidebar/guest-list/guest-template.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This template renders the RSVP ARI sidebar guest list item JS template.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/sidebar/guest-list/item-template.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+<script
+	class="tribe-tickets__rsvp-ar-guest-list-item--template"
+	id="tmpl-tribe-tickets__rsvp-ar-guest-list-item--template-<?php echo esc_attr( $rsvp->ID ); ?>"
+	type="text/template"
+>
+	<li
+		class="tribe-tickets__rsvp-ar-guest-list-item"
+		data-guest-number="{{data.attendee_id + 1}}"
+	>
+		<button
+			class="tribe-tickets__rsvp-ar-guest-list-item-button tribe-tickets__rsvp-ar-guest-list-item-button--inactive"
+			type="button"
+			data-guest-number="{{data.attendee_id + 1}}"
+		>
+			<?php $this->template( 'v2/components/icons/guest', [ 'classes' => [ 'tribe-tickets__rsvp-ar-guest-icon', 'tribe-tickets__rsvp-ar-guest-icon--inactive' ] ] ); ?>
+			<span class="tribe-tickets__rsvp-ar-guest-list-item-title tribe-tickets__rsvp-ar-guest-list-item-title--inactive tribe-common-a11y-visual-hide">
+				<?php
+					echo esc_html( tribe_get_guest_label_singular( 'RSVP attendee registration sidebar guest button' ) );
+				?>
+			</span>
```

Comment:
Since the button already has `tribe-tickets__rsvp-ar-guest-list-item-button--inactive`, I would base the styles of the inactive guest icon and title off of the button inactive class. If we have separate inactive classes for the button, icon, and title, we can technically have any combination of inactive an active for all 3 of these elements (though really it shouldn't). The best way to do this is to have it all based on one class, which determines whether the item is inactive or not.